src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -31,7 +31,7 @@
     <Compile Include="System\MarshalByRefObjectTest.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
-    <Compile Include="System\BitConverter.netcoreapp.cs" />
```

Comment:
The .netcoreapp file needs to be under this condition or else you'll fail netfx builds.